code/modules/clothing/head/misc.dm

```diff
@@ -333,10 +333,14 @@
 	desc = "Why not 'eagle head'? Who knows."
 	icon_state = "griffinhat"
 	item_state = "griffinhat"
-	flags = BLOCKHAIR|NODROP
+	flags = BLOCKHAIR
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE
 	action_button_name = "Caw"
 
+/obj/item/clothing/head/griffin/nodrop
+	..()
+	flags = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|NODROP

```

Comment:
you don't run `..()` for an object define. Also the flags you included here are `flags_inv` when you should only be `flags = BLOCKHAIR|NODROP`